pkg/hubble/parser/threefour/parser.go

```diff
@@ -15,7 +15,6 @@ import (
 	"github.com/sirupsen/logrus"
 	"google.golang.org/protobuf/types/known/wrapperspb"
 
-	"github.com/cilium/cilium/api/v1/flow"
 	pb "github.com/cilium/cilium/api/v1/flow"
```

Comment:
I think it's better to keep the unaliased version here maybe? `pb` is a pretty generic name that doesn't give much clue about what the package is actually doing.

Reply:
Unresolving this as it appears to still be using "pb", I agree with nick - pb might get confusing given there are multiple protobuffer libraries.